scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -1661,19 +1667,14 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
 
   def caseClause(): Case = atPos(in.prevTokenPos, auto) {
     require(token.isNot[`case`] && debug(token))
-    token match {
-      case token: Unquote if token.prev.is[Ellipsis] =>
-        unquote[Case]
-      case token =>
-        Case(pattern().require[Pat], guard(), {
-          accept[`=>`]
-          autoPos(blockStatSeq() match {
-            case List(q: impl.Quasi) => Term.Block(List(q))
-            case List(blk: Term.Block) => blk
-            case stats => Term.Block(stats)
-          })
-        })
-    }
+    Case(pattern().require[Pat], guard(), {
+      accept[`=>`]
+      autoPos(blockStatSeq() match {
+        case List(q: impl.Quasi) => Term.Block(List(q))
+        case List(blk: Term.Block) => blk
+        case stats => Term.Block(stats)
+      })
+    })
```

Comment:
That's a good change. I was looking forward to it since I put this change in place.